packages/flutter/lib/src/material/stepper.dart

```diff
@@ -204,12 +204,20 @@ class Stepper extends StatefulWidget {
     this.controlsBuilder,
     this.elevation,
     this.margin,
+    this.connectorColor,
+    this.connectorThickness = 2.0,
   }) : assert(steps != null),
        assert(type != null),
        assert(currentStep != null),
        assert(0 <= currentStep && currentStep < steps.length),
        super(key: key);
 
+  /// The thickness of the connecting lines.
+  final double? connectorThickness;
+
+  /// The color of the connection lines.
+  final Color? connectorColor;
```

Comment:
This should be a `MaterialStateProperty<Color?>?` so that you can use it to resolve the different colors for different states (e.g. `disabled`, in this case).